docker/dockerfile-python

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+MAINTAINER wxchan
```

Comment:
`MAINTAINER` is deprecated from v1.13.0.
https://github.com/docker/docker/releases/tag/v1.13.0